core/src/avm1/globals/as_broadcaster.rs

```diff
@@ -130,7 +130,12 @@ pub fn broadcast_internal<'gc>(
             let listener = listeners.array_element(i);
 
             if let Value::Object(listener) = listener {
-                listener.call_method(method_name, call_args, activation)?;
+                let (callback, base_proto) =
+                    search_prototype(Some(listener), method_name, activation, listener)?;
+
+                // .call does nothing if the method is undefined, as opposed to emitting a warning
+                // This is a good thing since not all listeners will have all methods
+                callback.call(method_name, activation, listener, base_proto, call_args)?;
```

Comment:
Using `call_method` is fine as the warning is only intended for debugging and hidden behind the `avm_debug` feature, so it doesn't get compiled in by default.

Reply:
Whoops, just noticed we warn in any case. However, let's still use `call_method` here (I'd prefer to squelch the warning inside call_method instead).

Reply:
I just reverted this, but I'm not sure I entirely agree. I feel like `call_method` _should_ warn because usually when you call a method, you're expecting it to be there, and if it isn't there, it's either a Ruffle issue or a bug in the SWF. That seems pretty useful for debugging. But with listeners, the broadcaster doesn't expect that every listener has a handler for every event; it only tries to call a handler if it exists. I think as more events are implemented (there are three more to implement just on TextField) the debug spam will get pretty intense, potentially drowning out more relevant warnings.

Perhaps it would make sense if there were an alternative to call_method which failed silently if no method was found? I'm not intimately familiar with the codebase yet, but I think there are plenty of other places where such a method would come in handy.

Thanks for your review!